WMF Framework/EventPlatformClientLibrary.swift

```diff
@@ -134,10 +126,21 @@ public class EPC: NSObject {
      */
     private let streamConfigServiceURI: URL
 
+    /**
+     * An individual stream's configuration.
+     */
+    private struct StreamConfiguration: Codable {
+        let sampling: Sampling?
+        struct Sampling: Codable {
+            let rate: Double?
+            let identifier: String?
+        }
+    }
```

Comment:
What would happen when deserializing to this if a stream's config included additional fields besides `sampling`?

Let's say the returned stream configurations suddenly look like this:

```json
{
  "streams": {
    "ios.edit_history_compare": {},
    "ios.daily_checkin": {
      "tags": { "anonymous": true }
    },
    "ios.some_old_stream": {
      "tags": { "active": false }
    }
  }
}
```

Reply:
In your example, the decoding would succeed, `tags` would be ignored and `sampling` would be `nil`.

Reply:
Awesome! :D